tests/e2e/specs/merchant/merchant-orders-partial-refund.spec.js

```diff
@@ -0,0 +1,197 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+const { merchant, shopper, uiUnblocked } = require( '@woocommerce/e2e-utils' );
+
+/**
+ * Internal dependencies
+ */
+import { merchantWCP } from '../../utils';
+import { fillCardDetails, setupProductCheckout } from '../../utils/payments';
+
+const card = config.get( 'cards.basic' );
+const product1 = config.get( 'products.simple.name' );
+const product2 = 'Belt';
```

Comment:
For my learning: where we have stored this data config?  Is it `sample_products_override.csv`?

Reply:
Thanks for reviewing, @naman03malhotra!

> where we have stored this data config

Every time we call `config.get('...')`, we're getting the object from the [test.json](https://github.com/Automattic/woocommerce-payments/blob/develop/tests/e2e/config/test.json) file.

As for the 'Belt' and the 'Hoodie', I just arbitrarily selected them from the pre-existing products in the store.